exercises/J-array-get-set/exercise.js

```diff
@@ -6,12 +6,18 @@
 
 function first(arr) {
   return; // complete this statement
+  return numbers[0];
```

Comment:
Hi Berenice,

I believe these return statements in line 8 and 14 may cause issues in your code. Once a return statement is run, it kicks out of the function or program. So it won't return line 9 or check  your conditional in line 14. if you remove line 8and 14 and always keep the return as the last statement you should be fine, or as in when you want the program to leave the function and return back to where it was called, or return a value.